CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Master
+------
+
+* Move SessionStorage from a generated class to a re-usable module. To
+  migrate to this new class delete the old generated SessionStorage class
+  under models and include the SessionStorage mixin in your Shop model
```

Comment:
This sentence is a bit awkward and refers to the new module as both a class and a mixin, maybe something like:

 - "To migrate to the new module, delete the old generated SessionStorage class in the models directory and include the SessionStorage module in your Shop model."